e2e/cp_test.go

```diff
@@ -58,57 +58,57 @@ func TestCopySingleS3ObjectToLocal(t *testing.T) {
 			name:        "cp s3://bucket/object .",
 			src:         "file1.txt",
 			dst:         ".",
-			expected:    fs.WithFile("file1.txt", fileContent, fs.WithMode(0644)),
+			expected:    fs.WithFile("file1.txt", fileContent, fs.WithMode(0o644)),
```

Comment:
Regarding these formatting changes, even though it's not important on this PR, it's better to decouple cosmetic changes with the actual changes(bugfix, feature etc.). No need to revert for this one.

Reply:
done.